internal/daemonset/daemonset.go

```diff
@@ -23,17 +21,14 @@ import (
 const (
 	kubeletDevicePluginsVolumeName = "kubelet-device-plugins"
 	kubeletDevicePluginsPath       = "/var/lib/kubelet/device-plugins"
-	nodeLibModulesVolumeName       = "node-lib-modules"
 	nodeVarLibFirmwarePath         = "/var/lib/firmware"
-	nodeVarLibFirmwareVolumeName   = "node-var-lib-firmware"
 )
 
 //go:generate mockgen -source=daemonset.go -package=daemonset -destination=mock_daemonset.go
 
 type DaemonSetCreator interface {
 	GarbageCollect(ctx context.Context, mod *kmmv1beta1.Module, existingDS []appsv1.DaemonSet, validKernels sets.Set[string]) ([]string, error)
```

Comment:
GarbageCollection can also be changed to deal only with DevicePlugin daemonsets

Reply:
Fixed